packages/hash/api/src/collab/Instance.ts

```diff
@@ -190,19 +182,21 @@ export class Instance {
     try {
       /**
        * This removes any extra properties from a passed object containing an
-       * accountId and entityId, which may be an Entity or a LatestEntityRef, or
+       * accountId and entityId, which may be an Entity or a LatestEntityRef,
+       * or
```

Comment:
This diff is probably caused by some local plugin in Nate’s editor. If the tool is available via CLI in some form (e.g. an eslint plugin), we can potentially enable it for everyone.

Personally, I don't bother with automatic word wrapping in comments because I see downsides in doing this. Adding a word at the beginning of a lengthy write-up can create a pretty large diff, similarly to what we see here. It may be also desirable to match the end of a sentence with the new line or draw a mini ASCII diagram, which an automatic rule can destroy.

Having said that, I’m open to trying something as a team, as long as we can sync the config amongst all devs and the CI.

Reply:
This is actually caused by a mismatch between editor config on my iMac vs Macbook. Will revert these changes, but will also track task about deciding whether to wrap commernts